addons/l10n_ar/__manifest__.py

```diff
@@ -4,34 +4,156 @@
+Functional
+----------
 
-* Activate ARS currency.
-* Create new model AFIP Responsibility to identify the partners fiscal responsibility.
-* Add the Identification Types valid in Argentina.
-* Define Argentinian chart of accounts:
+This module add accounting features for the Argentinian localization, which represent the minimal configuration needed for a company  to operate in Argentina and under the AFIP (Administración Federal de Ingresos Públicos) regulations and guidelines.
+
+Follow the next configuration steps for Production:
+
+1. Go to your company and configure your VAT number and AFIP Responsibility Type
+2. Go to Accounting / Settings and set the Chart of Account that you will like to use.
+3. Create your Sale journals taking into account AFIP POS info.
+
+Demo data for testing:
+
+* 3 companies were created, one for each AFIP responsibility type with the respective Chart of Account installed. Choose the company that fix you in order to make tests:
+
+  * (AR) Responsable Inscripto
+  * (AR) Exento
+  * (AR) Monotributo
+
+* Journal sales configured to Pre printed and Expo invoices in all companies
+* Invoices and other documents examples already validated in “(AR) Responsable Inscripto” company
+* Partners example for the different responsibility types:
+
+  * ADHOC (IVA Responsable Inscripto)
+  * Consejo Municipal Rosario (IVA Sujeto Exento)
+  * Gritti (Monotributo)
+  * Cerro Castor. IVA Liberado in Zona Franca
+  * Expresso (Cliente del Exterior)
+  * Odoo (Proveedor del Exterior)
+
+Highlights:
+
+* Chart of account will not be automatically installed, each CoA Template depends on the AFIP Responsibility of the company, you will need to install the CoA for your needs.
+* No sales journals will be generated when installing a CoA, you will need to configure your journals manually.
+* The Document type will be properly pre selected when creating an invoice depending on the fiscal responsibility of the issuer and receiver of the document and the related journal.
+* A CBU account type has been added and also CBU Validation
+
+
+Technical
+---------
+
+This module adds both models and fields that will be eventually used for the electronic invoice module. Here is a summary of the main features:
+
+Master Data:
+
+* Chart of Account: one for each AFIP responsibility that is related to a legal entity:
 
   * Responsable Inscripto (RI)
   * Exento (EX)
   * Monotributo (Mono)
 
-* Define Argentinian Taxes
-* Define Fiscal Positions
-* Define AFIP Legal Documents
-* Add AFIP Codes for models:
+* Argentinian Taxes and Account Tax Groups (VAT taxes with the existing aliquots and other types)
+* AFIP Responsibility Types
+* Fiscal Positions (in order to map taxes)
+* Legal Documents Types in Argentina
+* Identification Types valid in Argentina.
+* Country AFIP codes and Country VAT codes for legal entities, natural persons and others
+* Currency AFIP codes
+* Unit of measures AFIP codes
+* Partners: Consumidor Final and AFIP
+
+New Models:
```

Comment:
We should not mention all the new models/fields/menuitems in the manifest.  It is given automatically when you look at the module in debug mode in Odoo by the way and it occurred to me that the description is not even shown in the Apps in the Odoo.  (but does not mean we can not use it for the documentation later on) So from new Models should be removed.

Reply:
I understand. I think that we should remove only the menu items info but will be great to maintain the new models and fields. what do you think? This because after install a module and with a debug mode we can see the summary of the new menus and views created/inherit for the module, but we can not see the the list of fields and models created/inherit in the module 

Show to the user the information about the new models and fields I think is usefull in order to let him see the new functionalities, to understand the relevant fields and models for the localization.

NOTE: Apps in Odoo do you mean in https://apps.odoo.com? I have checked and I can confirm that the description of the manifest is showed there, but really no matter for us because all the odoo/odoo and odoo/enterprise modules are not part of the https://apps.odoo.com (i think):

Reply:
The new fields/models are not necessary either (and I think they are shown too).  I was talking about in the software itself for the apps.  Before, standard Odoo modules were also shown, but I don't know if it is still the case.

Reply:
Ok! in the Odoo software itself, well I am trying to tell you that in Odoo itself does not show the information about the module's fields and models. Only show the new menus/views/reports.

I leave you here a video to show you. https://drive.google.com/file/d/1S_XENGakqbJ-O0r-HQ6tH_WZWQZsaitg/view?usp=sharing

The description in manifest is showed in the "Information" Tab in the module form view., in the example of the video can be seen because the Project module has not description in manifest, but I leave here and screenshot about how the description is show in the model's form view for the l10n_ar

![image](https://user-images.githubusercontent.com/7593953/64034097-53668680-cb24-11e9-830e-93d36663d1e1.png)

Reply:
Ok, indeed, it is not included, but we don't want it in the description of the modules anyways.

Reply:
ok. removed